prophet/src/repositories.rs

```diff
@@ -0,0 +1,147 @@
+use std::path::{Path, PathBuf};
```

Comment:
I'm guessing this is where the code from `prophet/src/lib.rs` was moved to, making it a direct copy/paste?

Reply:
Yes.